iris_grib/_save_rules.py

```diff
@@ -1091,6 +1092,67 @@ def _cube_is_time_statistic(cube):
     return result
 
 
+def _cube_is_spatial_statistic(cube):
+    """
+    Test whether we can identify this cube as a statistic over space.
+
+    We need to know whether our cube represents a spatial statistic. This is
+    almost always captured in the cell methods.
+
+    WIP: UNFINISHED?
+    """
+    result = False
+    spatial_cell_methods = []
+    has_cell_methods = cube.cell_methods
+
+    if has_cell_methods:
+        spatial_cell_methods = [
+            cell_method for cell_method in cube.cell_methods if 'area' in
+            cell_method.coord_names]
+    else:
+        raise ValueError("")
+
+    if not spatial_cell_methods:
+        result = False
+        raise ValueError("Expected a cell method with a coordinate name "
+                         "of 'area'")
+
+    elif len(spatial_cell_methods) > 1:
+        result = False
+        raise ValueError("Cannot handle multiple 'area' cell methods")
+
+    elif len(spatial_cell_methods[0].coord_names) > 1:
+        result = False
+        raise ValueError("Cannot handle multiple coordinate names in "
+                         "the spatial processing related cell method. "
+                         "Expected ('area',), got {!r}".format(
+                             spatial_cell_methods[0].coord_names))
+
+    else:
+        result = True
+
+    return result
+
+
+def get_cell_method_name(cube, type):
+    """
+    Returns name of cell method in cube where cell_method.coord_names == type
+    """
+    pass
+
+
+def statistical_method_code(cell_method_name):
+    """
+    Decode cell_method string as statistic code integer.
+    """
+    statistic_code = _STATISTIC_TYPE_NAMES.get(cell_method_name, None)
+    if statistic_code is None:
+        msg = ('Product definition section 4 contains an unsupported '
+               'statistical process type [{}] ')
+        raise TranslationError(msg.format(statistic_code))
```

Comment:
Hmmm, don't know if PyCharm's just playing silly buggers but it doesn't recognize TranslationError in python 2.7.  Was it by any chance a thing that got developed for python 3?